src/Chapter1.hs

```diff
@@ -520,7 +522,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if abs(x) < abs(y) then x else y
```

Comment:
You can omit parentheses for the arguments 🙂 
Also, in Haskell functions and its arguments are always separated by spaces 👾 

```suggestion
closestToZero x y = if abs x < abs y then x else y
```